recipes/restinio/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RestinioConan(ConanFile):
+    name = "restinio"
+    license = "BSD-3-CLAUSE"
+    homepage = "https://github.com/Stiffstream/restinio"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "RESTinio is a header-only C++14 library that gives you an embedded HTTP/Websocket server."
+    topics = ("http-server", "websockets", "rest", "tls-support")
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
```suggestion
```

Reply:
At least the compiler is needed for the method: `tools.check_min_cppstd(self, "14")`.